cmake/CMakeLists.txt

```diff
@@ -11,6 +11,15 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 #set(CMAKE_VERBOSE_MAKEFILE ON)
 
+if(NOT DEFINED CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 11)
+  set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
If we wanted to use modern CMake:
> [This method isn't bad for a final package, but shouldn't be used by a library.](https://cliutils.gitlab.io/modern-cmake/chapters/features/cpp11.html)

the more modern alternative is available for CMake 3.8 and later, so after 2017, which counts as quite recent in my book. I'm usually not a fan of pushing recent CMake versions because it's been an issue for me in the past, so I don't recommend this change now, but I'll keep an eye on this question.

Reply:
Well, we _are_ already requiring CMake 3.13 since 60f6bc42 (added because you started creating symlinks in #539).

Reply:
Update: I tried the more modern suggestion from that link, and it didn't allow specifying a later C++ version. I think that recipe is good for choosing a specific version for your package. However, we're not the ones deciding it and we rather need an easy way to adopt other people's choices during testing.

Leaving the `CMAKE_CXX_STANDARD` in place for now until a better solution is available.